src/EdFi.AnalyticsMiddleTier.DataStandard2/RLS/MSSQL/0010-View-rls_UserStudentDataAuthorization-Update.sql

```diff
@@ -0,0 +1,103 @@
+     INNER JOIN
+         analytics_config.DescriptorConstant
+         ON DescriptorMap.DescriptorConstantId = DescriptorConstant.DescriptorConstantId
+     INNER JOIN
+         edfi.StudentSchoolAssociation
+         ON StaffEducationOrganizationAssignmentAssociation.EducationOrganizationId = StudentSchoolAssociation.SchoolId
+     INNER JOIN
+         edfi.Student
+         ON Student.StudentUSI = StudentSchoolAssociation.StudentUSI
+     WHERE DescriptorConstant.ConstantName = 'AuthorizationScope.School'
+           AND StaffEducationOrganizationAssignmentAssociation.EndDate IS NULL
+           AND StudentSchoolAssociation.ExitWithdrawDate IS NULL
+     UNION ALL
+
+     -- distinct because a student could be in two sections taught by same teacher
+     SELECT DISTINCT 
+         Staff.StaffUniqueId AS UserKey, 
+         Student.StudentUniqueId AS StudentKey
+     FROM 
+         edfi.Staff
+     INNER JOIN
+         edfi.StaffEducationOrganizationAssignmentAssociation
+         ON Staff.StaffUSI = StaffEducationOrganizationAssignmentAssociation.StaffUSI
+     INNER JOIN
+         analytics_config.DescriptorMap
+         ON StaffEducationOrganizationAssignmentAssociation.StaffClassificationDescriptorId = DescriptorMap.DescriptorId
+     INNER JOIN
+         analytics_config.DescriptorConstant
+         ON DescriptorMap.DescriptorConstantId = DescriptorConstant.DescriptorConstantId
+     INNER JOIN
+         edfi.StaffSectionAssociation
+         ON StaffEducationOrganizationAssignmentAssociation.StaffUSI = StaffSectionAssociation.StaffUSI
+            AND StaffEducationOrganizationAssignmentAssociation.EducationOrganizationId = StaffSectionAssociation.SchoolId
+     INNER JOIN
+         edfi.StudentSectionAssociation
+         ON StudentSectionAssociation.SchoolId = StaffSectionAssociation.SchoolId
+            AND StudentSectionAssociation.ClassPeriodName = StaffSectionAssociation.ClassPeriodName
+            AND StudentSectionAssociation.ClassroomIdentificationCode = StaffSectionAssociation.ClassroomIdentificationCode
+            AND StudentSectionAssociation.LocalCourseCode = StaffSectionAssociation.LocalCourseCode
+            AND StudentSectionAssociation.TermDescriptorId = StaffSectionAssociation.TermDescriptorId
+            AND StudentSectionAssociation.SchoolYear = StaffSectionAssociation.SchoolYear
+            AND StudentSectionAssociation.UniqueSectionCode = StaffSectionAssociation.UniqueSectionCode
+            AND StudentSectionAssociation.SequenceOfCourse = StaffSectionAssociation.SequenceOfCourse
+     INNER JOIN
+         edfi.Student
+         ON StudentSectionAssociation.StudentUSI = Student.StudentUSI
+     WHERE DescriptorConstant.ConstantName = 'AuthorizationScope.Section'
+           AND StaffEducationOrganizationAssignmentAssociation.EndDate IS NULL
+           AND (StudentSectionAssociation.EndDate IS NULL OR StudentSectionAssociation.EndDate >= GETDATE());
```

Comment:
Tests should fail if I remove StudentSectionAssociation.EndDate >= GETDATE(), but they didn't fail.